app/lib/app/SecvisogramPage/View/shared/HTMLTemplate/Template.html

```diff
@@ -331,7 +331,7 @@ <h2>Sharing rules</h2>
     <p>
       {{#tlp}}
         <b>{{#label}}TLP:{{.}}{{/label}}</b><br>
-        For the TLP version see: <a href={{#url}}{{.}}{{/url}}{{^url}}https://www.first.org/tlp/{{/url}}>{{#url}}{{.}}{{/url}}{{^url}}https://www.first.org/tlp/{{/url}}</a>
+        For the TLP version see: <a {{#secureHref}}{{#url}}{{.}}{{/url}}{{/secureHref}}{{^url}}https://www.first.org/tlp/{{/url}}>{{#url}}{{.}}{{/url}}{{^url}}https://www.first.org/tlp/{{/url}}</a>
```

Comment:
Actually, the current implementation breaks the default value (`https://www.first.org/tlp/`) that is inserted if no `url` is given. I guess the following might fix it:
```suggestion
        For the TLP version see: <a {{#secureHref}}{{#url}}{{.}}{{/url}}{{/secureHref}}{{^url}}href=https://www.first.org/tlp/{{/url}}>{{#url}}{{.}}{{/url}}{{^url}}https://www.first.org/tlp/{{/url}}</a>